council/gitflow.md

```diff
@@ -0,0 +1,243 @@
+and attention.
+
+## Hotfix branches
+
+Hotfix branches are very much like release branches in that they are also meant to 
+prepare for a new production release, albeit unplanned. They arise from the necessity 
+to act immediately upon an undesired state of a live production version. When a 
+critical bug in a production version must be resolved immediately, a hotfix branch may 
+be branched off from the corresponding tag on the master branch that marks the 
+production version.
+
+![Hotfix branches](img/hotfix-branches.png "Hotfix branches")
+
+The essence is that work of team members (on the develop branch) can continue, while 
+another person is preparing a quick production fix.
+
+Hotfix branches are created from the master branch. For example, say version 1.2 is 
+the current production release running live and causing troubles due to a severe bug. 
+But changes on develop are yet unstable. We may then branch off a hotfix branch and 
+start fixing the problem:
+
+```bash
+# certify you are at the master branch
+$ git status                       
+On branch master
+
+# update branch
+$ git fetch --all --prune
+$ git pull origin master
+
+# create hotfix branch
+$ git checkout -b hotfix/1.2.1
+
+# Update and add files, bump version and commit
+git commit -m "Fix something. Bumped version number to 1.2.1"
+
+# push new branch and open a pull request pointing to master
+$ git push origin hotfix/1.2.1
+```
+
+After merge with *master* you need to update the *develop* branch with the hotfix too
+to avoid conflicts and loose code. 
+
+```bash
+$ git checkout develop
+Switched to branch 'develop'
+
+$ git merge --no-ff hotfix/1.2.1
+Merge made by recursive.
+(Summary of changes)
+```
+
+Hotfix branches should always branch off from `master` branch and must merge back into
+`master` too. Also this should be merged in `develop` too. The branch naming convention 
+for hotfix branches should be anything like `hotfix/something`, where something can be 
+anything except master, develop, release-*, or hotfix-*.
+
+## Release branches
+
+Release branches support preparation of a new production release. They allow for 
+last-minute dotting of i’s and crossing t’s. Furthermore, they allow for minor bug fixes 
+and preparing meta-data for a release (version number, build dates, etc.). By doing all 
+of this work on a release branch, the develop branch is cleared to receive features for 
+the next big release.
+
+The key moment to branch off a new release branch from develop is when develop (almost) 
+reflects the desired state of the new release. At least all features that are targeted 
+for the release-to-be-built must be merged in to develop at this point in time. All 
+features targeted at future releases may not—they must wait until after the release 
+branch is branched off.
+
+It is exactly at the start of a release branch that the upcoming release gets assigned 
+a version number—not any earlier. Up until that moment, the develop branch reflected 
```

Comment:
I'd specify that the version number will be sufixed with a `-rcX`, example: `v1.2.0-rc5`. Subsequent commits will increase the X in `-rcX`, although maybe not all commits must include the tag. At the end of the day those tags are used to indicate the QA team that "this code should be tested to check if it's production ready"